pkg/k8s/secrets.go

```diff
@@ -97,9 +97,16 @@ func (c secretClient) Create(secret types.Secret) error {
 				secretLabel: secretLabelValue,
 			},
 		},
-		StringData: map[string]string{
-			secret.Name: secret.Value,
-		},
+	}
+
+	if len(secret.RawValue) > 0 {
+		req.Data = map[string][]byte{
+			secret.Name: secret.RawValue,
```

Comment:
have you manually tested this, i know that when you get the value from the k8s API, these values are base64 encoded, so i don't know if we need to send them already encoded or if it accepts the non-encoded values during creation.

Reply:
I'll try that.

Reply:
one other things, i think that we _need_ to require the `secret.RawValue` to be base64 encoded when it is submitted to faas-netes, this ensures that the payload is parsed correctly. Which means that either we need to decode it so that we can submit it raw  _or_ because it is already encoded, we wouldn't need to do anything and the code here would work as expected.

Reply:
Seems to work as expected for the existing path.

Reply:
I've done a couple of manual tests and updated the description. Let me know what you think.